<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlogiqueWidget</class>
 <widget class="QWidget" name="BlogiqueWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Subject_"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="PostFrame_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDockWidget" name="SideWidget_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="windowTitle">
      <string notr="true">Blogique</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolBox" name="Tools_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="BlogPosts_">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>369</width>
            <height>490</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Blog posts</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="SaveEntry_">
   <property name="text">
    <string>Save as draft</string>
   </property>
  </action>
  <action name="OpenInBrowser_">
   <property name="text">
    <string>Open in browser</string>
   </property>
  </action>
  <action name="Submit_">
   <property name="text">
    <string>Submit</string>
   </property>
  </action>
  <action name="UpdateProfile_">
   <property name="text">
    <string>Update profile</string>
   </property>
  </action>
  <action name="SaveNewEntry_">
   <property name="text">
    <string>Save as a new draft</string>
   </property>
   <property name="toolTip">
    <string>Save entry as a new draft</string>
   </property>
  </action>
  <action name="NewEntry_">
   <property name="text">
    <string>New post</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
